code/modules/awaymissions/gateway.dm

```diff
@@ -266,7 +272,10 @@ obj/machinery/gateway/centerstation/process()
 			calibrated = 1
 			return
 
-/obj/machinery/gateway/centerstation/attack_ghost(mob/user as mob)
+/obj/machinery/gateway/centerstation/attack_ghost(mob/user)
```

Comment:
did you test this? pretty sure aghosts call attack_ai when they click things, not attack_ghost

Reply:
I have now and I can confirm it works